### ISSUE TITLE
The way gets image info could be broken.

### ISSUE THREAD

**Issue by luwensu** [jramseye/docker-migrate#6]
Once a image `CREATED` includes something like
`About one hour ago`, it may brake `if (i % 8 == 0):`

**Comment by shishir-a412ed** [comment]
@luwensu 
Yes I am working on a fix for this.`unix awk` would be better for this situation.
You can follow the work on this project over here.

https://github.com/shishir-a412ed/Docker-Migrate
